src/platform/silabs/SiWx917/CHIPCryptoPALTinyCrypt.cpp

```diff
@@ -926,20 +920,18 @@ CHIP_ERROR Spake2p_P256_SHA256_HKDF_HMAC::MacVerify(const uint8_t * key, size_t
 CHIP_ERROR Spake2p_P256_SHA256_HKDF_HMAC::FELoad(const uint8_t * in, size_t in_len, void * fe)
 {
     CHIP_ERROR error = CHIP_NO_ERROR;
-    int result       = 0;
 
     uECC_word_t tmp[2 * NUM_ECC_WORDS] = { 0 };
     uECC_vli_bytesToNative(tmp, in, NUM_ECC_BYTES);
 
     uECC_vli_mmod((uECC_word_t *) fe, tmp, curve_n);
 
-exit:
-    _log_mbedTLS_error(result);
     return error;
 }
 
 CHIP_ERROR Spake2p_P256_SHA256_HKDF_HMAC::FEWrite(const void * fe, uint8_t * out, size_t out_len)
 {
+    (void) out_len;
```

Comment:
there is no LHS in this statement .

You are typecasting but it should assign to something otherwise there is no meaning of this statement

Reply:
getting unused variable warning for **out_len** since we are not using it.
to overcome that added **(void) out_len;**